Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
```

Comment:
We started following the convention to make the minor number match the sprint. Please update version to match .1.123.0

Reply:
Fixed.